src/Persistence/YamlValidator/SchemaLoader.cs

```diff
@@ -0,0 +1,49 @@
+using System.Reflection;
+using Json.Schema;
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.YamlValidator;
+
+public class SchemaLoader
+{
+    private const string _schemaFolderPath = "schema";
+    private const string _subschemaFolderPath = "subschemas";
+
+    public JsonSchema Load()
+    {
+        var assembly = Assembly.GetExecutingAssembly();
```

Comment:
due to performance issues, do not use `Assembly.GetExecutingAssembly()`. When getting types etc, use `typeof(SomeType)` when the type is known statically, or `value.GetType()` when only known at runtime.

Reply:
you can use `typeof(SomeType).Assembly`